overlay/osx/freetype/CONTROL

```diff
@@ -2,7 +2,7 @@ Source: freetype
 Version: 2.10.4
 Homepage: https://www.freetype.org/
 Description: A library to render fonts.
-Default-Features: zlib, bzip2, png, brotli
+Default-Features: zlib, bzip2, png
```

Comment:
A port should not be in the overlay just to select features; that is the point of features. Instead, explicitly set the features by installing `freetype[zlib, bzip2, png]`.

Reply:
I have tried that, but it is ignored.
It is because freetype is include as requiremt with it's default options by other libraries.

Reply:
Ah, that makes sense. Which libraries depend on it? If those libraries are in our overlay anyway, we can edit their vcpkg.json manifests to select the features for freetype.

Reply:
Unfortunately not is is included by many for instance harfbuz

Reply:
I'm confused. What is not included?

Reply:
The patch was merged upstream:
https://github.com/microsoft/vcpkg/pull/19444

Reply:
I'm confused what that PR has to do with selecting the features for freetype.

Reply:
Nothing. That was the reason why harfbuz is no longer in our overlay

Reply:
Oh, you mean the libraries that depend on freetype are not in the overlay, right? Is freetype is a dependency of a dependency of Qt? Am I understanding that right?

Reply:
Yes

Reply:
Okay. Please explain in the commit message that we cannot change the port features of the package requiring this because this is a dependency of a dependency of Qt.

Reply:
Done.